package.json

```diff
@@ -102,12 +112,12 @@
     "flow": "flow",
     "flow-typed-rebuild": "rm -rf flow-typed/ && flow-typed install && flow-typed install --rootDir packages/@cruise-automation/button && flow-typed install --rootDir packages/@cruise-automation/tooltip && flow-typed install --rootDir packages/react-key-listener && flow-typed install --rootDir packages/regl-worldview",
     "docs": "webpack-dev-server",
-    "docs-deploy": "gh-pages -d docs/public",
+    "docs-deploy": "cp -r docs/public __temp_deploy__ && cp -f packages/webviz-core/public/index.html __temp_deploy__/index.html && cp -f packages/webviz-core/node_modules/wasm-lz4/wasm-lz4.wasm __temp_deploy__/dist/wasm-lz4.wasm && gh-pages -d __temp_deploy__ && rm -rf __temp_deploy__",
     "publish": "lerna run clean && lerna run build && lerna publish",
-    "screenshot": "NODE_ENV=development storybook-chrome-screenshot -c stories --parallel 8 --inject-files stories/injected.js",
+    "screenshot": "NODE_ENV=development storybook-chrome-screenshot -c stories --parallel 8 --inject-files stories/injected.js --browser-timeout 60000",
     "screenshot-ci": "npm run screenshot -- --puppeteer-launch-config '{\"executablePath\": \"/usr/bin/google-chrome\", \"headless\": false, \"args\":[\"--use-gl=swiftshader\", \"--no-sandbox\",\"--disable-setuid-sandbox\", \"--disable-dev-shm-usage\", \"--headless\", \"--mute-audio\"]}'",
     "screenshot-debug": "npm run screenshot -- --parallel 1 --debug --puppeteer-launch-config '{\"headless\": false}'",
-    "ci": "npm run install-ci && npm run lint && npm run flow && npm test && npm run build && npm run screenshot-ci"
+    "ci": "npm run install-ci && npm run build && npm run lint && npm run flow && npm test && npm run screenshot-ci"
```

Comment:
Have to do build before everything else, otherwise lint/flow etc cannot resolve internal packages.